Cannot build source code, com.tobiasdiez.easybind.EasyBind.listBind doesn't exist

- [yes] : I have tested the latest development version from http://builds.jabref.org/master/ and the problem persists

<!-- Add a clear and concise description of what the bug is. -->

Steps to reproduce the behavior:
1. clone JabRef github repository
2. try to gradle build
3. build fails: jabref/src/main/java/org/jabref/gui/JabRefFrame.java:992: error: cannot find symbol
            EasyBind.listBind(taskProgressView.getTasks(), stateManager.getBackgroundTasks());

It looks like the dependency has changed and listBind doesn't exist anymore

--- a/src/main/java/org/jabref/gui/JabRefDialogService.java
+++ b/src/main/java/org/jabref/gui/JabRefDialogService.java
@@ -294,7 +294,7 @@ public class JabRefDialogService implements DialogService {
     @Override
     public <V> Optional<ButtonType> showBackgroundProgressDialogAndWait(String title, String content, StateManager stateManager) {
         TaskProgressView<Task<?>> taskProgressView = new TaskProgressView<>();
-        EasyBind.listBind(taskProgressView.getTasks(), stateManager.getBackgroundTasks());
+        EasyBind.bindContent(taskProgressView.getTasks(), stateManager.getBackgroundTasks());
         taskProgressView.setRetainTasks(false);
         taskProgressView.setGraphicFactory(BackgroundTask::getIcon);
 

--- a/src/main/java/org/jabref/gui/JabRefFrame.java
+++ b/src/main/java/org/jabref/gui/JabRefFrame.java
@@ -1005,7 +1005,7 @@ public class JabRefFrame extends BorderPane {
 
         indicator.setOnMouseClicked(event -> {
             TaskProgressView<Task<?>> taskProgressView = new TaskProgressView<>();
-            EasyBind.listBind(taskProgressView.getTasks(), stateManager.getBackgroundTasks());
+            EasyBind.bindContent(taskProgressView.getTasks(), stateManager.getBackgroundTasks());
             taskProgressView.setRetainTasks(true);
             taskProgressView.setGraphicFactory(BackgroundTask::getIcon);
 

--- a/src/main/java/org/jabref/gui/exporter/ExportCustomizationDialogView.java
+++ b/src/main/java/org/jabref/gui/exporter/ExportCustomizationDialogView.java
@@ -55,7 +55,7 @@ public class ExportCustomizationDialogView extends BaseDialog<Void> {
         viewModel = new ExportCustomizationDialogViewModel(preferences, dialogService, repository);
         exporterTable.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
         exporterTable.itemsProperty().bind(viewModel.exportersProperty());
-        EasyBind.listBind(viewModel.selectedExportersProperty(), exporterTable.getSelectionModel().getSelectedItems());
+        EasyBind.bindContent(viewModel.selectedExportersProperty(), exporterTable.getSelectionModel().getSelectedItems());
         nameColumn.setCellValueFactory(cellData -> cellData.getValue().name());
         layoutColumn.setCellValueFactory(cellData -> cellData.getValue().layoutFileName());
         extensionColumn.setCellValueFactory(cellData -> cellData.getValue().extension());

--- a/src/main/java/org/jabref/gui/importer/ImportCustomizationDialog.java
+++ b/src/main/java/org/jabref/gui/importer/ImportCustomizationDialog.java
@@ -59,7 +59,7 @@ public class ImportCustomizationDialog extends BaseDialog<Void> {
         viewModel = new ImportCustomizationDialogViewModel(preferences, dialogService);
         importerTable.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
         importerTable.itemsProperty().bind(viewModel.importersProperty());
-        EasyBind.listBind(viewModel.selectedImportersProperty(), importerTable.getSelectionModel().getSelectedItems());
+        EasyBind.bindContent(viewModel.selectedImportersProperty(), importerTable.getSelectionModel().getSelectedItems());
         nameColumn.setCellValueFactory(cellData -> new ReadOnlyStringWrapper(cellData.getValue().getName()));
         classColumn.setCellValueFactory(cellData -> new ReadOnlyStringWrapper(cellData.getValue().getClassName()));
         basePathColumn.setCellValueFactory(cellData -> new ReadOnlyStringWrapper(cellData.getValue().getBasePath().toString()));

--- a/src/main/java/org/jabref/gui/texparser/ParseLatexDialogView.java
+++ b/src/main/java/org/jabref/gui/texparser/ParseLatexDialogView.java
@@ -76,7 +76,7 @@ public class ParseLatexDialogView extends BaseDialog<Void> {
         EasyBind.subscribe(fileTreeView.rootProperty(), root -> {
             ((CheckBoxTreeItem<FileNodeViewModel>) root).setSelected(true);
             root.setExpanded(true);
-            EasyBind.listBind(viewModel.getCheckedFileList(), fileTreeView.getCheckModel().getCheckedItems());
+            EasyBind.bindContent(viewModel.getCheckedFileList(), fileTreeView.getCheckModel().getCheckedItems());
         });
 
         latexDirectoryField.textProperty().bindBidirectional(viewModel.latexFileDirectoryProperty());

